### DISCUSSION TITLE
Temporal force-directed with less wiggling of nodes.

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [d3/d3#3931] [Q&A]
@mbostock Hi Mike I changed the radius of node 1549 so I could monitor it better over time. I was wondering if it is possible to keep it less jumping around, more static position?

[1]
https://observablehq.com/@d3/temporal-force-directed-graph

**Comment by Fil** [comment]
You can use fx and fy to fix the position of a node. See https://d3js.org/d3-force/simulation#simulation_nodes

**Reply by f1-outsourcing** [reply]
Ok so there is not some default functionality that:

- node first printed + (n other nodes, relationships)
- record currently assigned x,y for class nodes for history period t
- graph re printed.
- returning nodes are checked for history x,y
  some magic code that checks what is the most optimal solution is to print known nodes at their previous location and still maintain some normal graph like representation with new nodes and relationships.
